README.md

```diff
@@ -52,15 +52,4 @@ qiime emperor plot \
     --m-metadata-file sample-metadata.tsv \
     --o-visualization umap-emperor.qzv
     
-# optionally
-qiime umap center \
-  --i-embedding jaccard_umap.qza \
-  --o-centered-embedding centered_jaccard.qza
-  
-qiime emperor plot \
```

Comment:
Why remove the emperor example?

Reply:
this only removes the emperor example on the manually centered data. See line 50 of this file for another emperor visualization following call to `qiime umap embed`